model/role/db.go

```diff
@@ -0,0 +1,39 @@
+package role
+
+import (
+	"github.com/evergreen-ci/evergreen/db"
+	"github.com/mongodb/anser/bsonutil"
+	adb "github.com/mongodb/anser/db"
```

Comment:
is it, in anyway viable to avoid using the shim, and just using the new driver directly?

Reply:
Not without modifying the upsert utility function we have, which will require refactoring a number of places. I'm for it, but I don't think this is the place for that

Reply:
If not here then where? you can still upsert in other places...

Reply:
A separate ticket to refactor the upsert function in the db package. It currently uses the anser wrapper